### DISCUSSION TITLE
[DEV] change config table, column to row ?

### DISCUSSION THREAD

**Discussion by abarrau** [magicbug/Cloudlog#2708] [General]
Hello All Cloudlog Dev teams,
I see the config db table data are in column and not in row ? (like options table). 
Perhaps it's better to transform this table as like options ; for integrate more config data ? 
what do you think ? 
I can work on this change, if you want
73, Auré

**Comment by AndreasK79** [comment]
Hi Auré.

That was how it was made a long time ago. I say leave it. Why fix something that works?
Instead, if you want to add new features, use options or user_options now that these tables exists.

**Comment by abarrau** [comment]
Ok @AndreasK79 , if the new configuration are on options table, no problem. 
my proposal was to standardize the code ;)
If i would add a configuration value (global), i will use the options table ; tnx !
